src/components/User/OfflineOperations/Preissuance/components/PreissuanceForm.vue

```diff
@@ -227,9 +226,6 @@ export default {
             globalize('preissuance-form.asset-not-found',
               { assetCode: assetCode }
             )
-          ),
-          globalize('preissuance-form.asset-not-found',
```

Comment:
Why did you remove this? I know second param in ErrorHandler.process is translate id, but now the user will have notification without msg

Reply:
fixed